http-server-netty/src/test/groovy/io/micronaut/http/server/netty/http2/H2cSpec.groovy

```diff
@@ -229,6 +229,72 @@ class H2cSpec extends Specification {
         content.release()
     }
 
+    def 'prior knowledge'() {
```

Comment:
I don't understand this test, but that's my failing, not yours 😉

Reply:
\~~netty magic~~